lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -3793,14 +3793,24 @@ class ConvertAtenPoolingBaseOp : public OpConversionPattern<AtenOpT> {
 
   static int64_t getOutputDim(int64_t inputDim, int64_t kernelDim,
                               int64_t stride, int64_t padBefore,
-                              int64_t padAfter, int64_t dilation) {
+                              int64_t padAfter, int64_t dilation,
+                              bool ceilMode=false) {
     if (inputDim == kUnknownSize) {
       return kUnknownSize;
     } else {
-      return (
-          (inputDim + padBefore + padAfter - dilation * (kernelDim - 1) - 1) /
-              stride +
-          1);
+      int64_t dimSize = inputDim + padBefore + padAfter - dilation * (kernelDim - 1) - 1;
+      if (!ceilMode) {
+        dimSize = (dimSize / stride + 1);
+      }
+      else {
+        if (dimSize % stride == 0) {
+          dimSize = (dimSize / stride + 1);
+        }
+        else {
+          dimSize = (dimSize / stride + 2);
+        }
+      }
+      return dimSize;
```

Comment:
I somehow lost this comment - not sure where it went.
```suggestion
      int64_t dimSize = inputDim + padBefore + padAfter - dilation * (kernelDim - 1) - 1;
      if (ceilMode && (dimSize % stride != 0))
        return dimSize / stride + 2;
      return dimSize / stride + 1;
```